plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -36,6 +37,8 @@ fu_plugin_thunderbolt_device_created(FuPlugin *plugin, FuDevice *dev, GError **e
 	fu_plugin_add_rule(plugin,
 			   FU_PLUGIN_RULE_INHIBITS_IDLE,
 			   "thunderbolt requires device wakeup");
+	if (fu_plugin_get_config_value_boolean(plugin, "RetimerOfflineMode"))
+		fu_device_set_private_flags(dev, FU_THUNDERBOLT_DEVICE_FLAG_FORCE_ENUMERATION);
```

Comment:
fu_device_add_private_flag no?

Reply:
ups overlooked.... sorry for the delay I had to setup my github access in another laptop